DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: epiworldR
 Type: Package
 Title: Fast Agent-Based Epi Models
-Version: 0.6-0
+Version: 0.6-0.1
```

Comment:
I wonder if we want to start from 0. Also, I realize that I don't like the dash in the version, but I think CRAN still turns this into a dash, so this is fine (random comment!).

Reply:
We can start increasing the last digit the next that that there's only changes in the R side.

Reply:
Sounds good!